lib/internal/test_runner/utils.js

```diff
@@ -54,7 +54,8 @@ const kMultipleCallbackInvocations = 'multipleCallbackInvocations';
 const kRegExpPattern = /^\/(.*)\/([a-z]*)$/;
 
 const kPatterns = ['test', 'test/**/*', 'test-*', '*[._-]test'];
-const kDefaultPattern = `**/{${ArrayPrototypeJoin(kPatterns, ',')}}.?(c|m)js`;
+const kFileExtension = getOptionValue('--experimental-strip-types') ? '(c|m)(t|j)s' : '(c|m)js';
```

Comment:
I think we should make all of the supported file extensions explicit (ie `cjs|mjs|js`, etc.). Looking at this current change, it seems like `.s` files might be supported, but it's not 100% clear.

Reply:
Yes, that's a better idea. I'll change it later today.

`cjs` `mjs` `js` `ts` `mts` `cts`